docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -65,6 +142,10 @@ A value enclosed in single quotes results in a [JsonException](xref:System.Text.
 ''' is an invalid start of a value.
 ```
 
+### Maximum depth
```

Comment:
To match the table ordering, move this above `JSON strings (property names and string values)`